src/pages/AdminTemplates/AdminTemplates.scss

```diff
@@ -0,0 +1,169 @@
+@import "scss/constants.scss";
+
+.templates-page {
+  display: flex;
+  flex: 1;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  padding-top: 5rem;
+
+  &__nav-buttons {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    width: 80%;
+
+    &--cancel {
+      padding: 0.5rem 1.5rem;
+      border-radius: 22px;
+      background-color: lighten($color: $saturated-black, $amount: 10);
+      color: $white;
+      border: none;
+
+      &:hover {
+        background-color: lighten($color: $saturated-black, $amount: 20);
+      }
```

Comment:
Instead of inlining these `lighten` calls all across the app, it might be a better idea to define them in `scss/constants.scss`, perhaps named something like:

- `$saturated-black--lighten-10`
- `$saturated-black--lighten-20`
- etc

(this applies to all cases where this is used, though I'm only calling it out once here)

Reply:
Also, what are the extra `$color` and `$amount` in this call for? In the docs, that is just describing the positional parameters, it doesn't use named params:

- https://sass-lang.com/documentation/modules/color#lighten

So this should just be `lighten($saturated-black, 10)`, etc